module-2/backend/week_one.md

```diff
@@ -1,32 +1,151 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+1. List the five common HTTP verbs and what the purpose is of each verb.
 
-Note: When you're done, submit a PR. 
+* **GET POST PUT PATCH DELETE**
+* GET is used to read or inspect a ceratin endpoint or resource
+* POST is used to create a resourse at a specific URI
+* PUT is used to edit an entire resource at a specific URI
+* PATCH is used to edit a specific part of a resource at a URI
+* DELETE destroys a resource at a certain URI
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
 2. What is Sinatra?
+
+* Sinatra is an ultra light, rack based, ruby specific, MVC, web application framework
+
 4. What is MVC?
+
+* MVC stands for model view controller. This is an architectural pattern used to design applications. It breaks the application into three seperate logical components.
+* The model component handles all data related logic that the user works with. It also acts as a go between for the DB and the controller.
+* The view component is used for all the UI and UI logic of the app.
+* The controller acts as an interface between the model and the views. Routes and URIs are defined here. The controller also allocates resources to certain views.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+* So that our team does not hate us. Also, following predefined schemas and patterns helps the team work in unison, with a clear goal, and with clear expectations and procedures. Furthermore, convention dictates WHERE code should live. And this makes navigating a codebase easier.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+

```

Comment:
looking for instance variables but this is true. PS dont do this, push this down the stack.